docs/tutorials/mobile/index.md

```diff
@@ -9,73 +9,103 @@ redirect_from: /get-started/with-mobile
+* All targets have support for CPU and this is the default
+* Applications that run on Android also have support for NNAPI and XNNPACK
+* Applications that run on iOS also have support for CoreML and XNNPACK
 
-   You need to understand your mobile app's scenario and get an ONNX model that is appropriate for that scenario. For example, does the app classify images, do object detection in a video stream, summarize or predict text, or do numerical prediction.
+Accelerators are called Execution Providers in ONNX Runtime.
 
-   ONNX models can be obtained from the [ONNX model zoo](https://github.com/onnx/models), converted from PyTorch or TensorFlow, and many other places.
+If the model is quantized, start with the CPU Execution Provider. If the model is not quantized start with XNNPACK. These are the simplest and most consistent as everything is running on CPU.
 
-   Once you have sourced or converted the model into ONNX format, it must be [converted to an ORT format model](../../reference/ort-format-models.md#convert-onnx-models-to-ort-format) in order to be used with the ONNX Runtime mobile package. This conversion is not necessary if you are using the full package.
+If CPU/XNNPACK do not meet the application's performance results, then try NNAPI/CoreML. Performance with these execution providers is device and model specific. If the model is broken into multiple partitions due to the model using operators that ONNX Runtime, NNAPI/CoreML. or the device doesn't support (e.g. older NNAPI versions), performance may degrade.
 
-3. How do I bootstrap my app development?
+Specific execution providers are configured in the SessionOptions, when the ONNXRuntime session is created and the model is loaded. For more detail, see your language [API docs](../../api).
 
-   If you are starting from scratch, bootstrap your mobile application according in your mobile framework XCode or Android Development Kit. TODO check this.
+### Measure the application's performance
 
-   a. Add the ONNX Runtime dependency
-  
-   b. Consume the onnxruntime API in your application
-   
-   c. Add pre and post processing appropriate to your application and model
+Measure the application's performance against the requirements of your target platform. This includes:
 
-4. How do I optimize my application?
+* application binary size
+* model size
+* application latency
+* power consumption
 
-   **To reduce binary size:**  Use the ONNX Runtime mobile package or a custom build to reduce the binary size. The mobile package requires use of an ORT format model.
+If the application does not meet its requirements, there are optimizations that can be applied.
 
-   **To reduce memory usage:** Use an ORT format model as that uses less memory.
+### Optimize your application
+
+#### Reduce model size
+
+One method of reducing model size is to quantize the model. This reduces an original model with 32-bit weights by approximately a factor of 4, as the weights are reduced to 8-bit.
```

Comment:
Should this link to ORT doco on quantization? We're telling people what they can do but not how they would do it.

Reply:
Yes